src/content/docs/infrastructure/install-infrastructure-agent/configuration/infrastructure-agent-configuration-settings.mdx

```diff
@@ -3483,6 +3485,98 @@ Metrics can also be enriched with extended cloud metadata (including custom reso
     </table>
   </Collapser>
 
+  <Collapser
+    className="freq-link"
+    id="exclude-matching-metrics"
+    title="exclude_matching_metrics"
+  >
+    <Callout variant="important">
+      Currently, this setting only applies to an operating system's processes metrics.
+    </Callout>
+
+    You can control how much data is sent to New Relic by configuring `exclude_matching_metrics`, which allows you to restrict the transmission of metric data based on the values of metric [attributes](/docs/nrql/using-nrql/query-infrastructure-dimensional-metrics-nrql/#naming-conventions).
+
+    You exclude metric data by defining literal or partial values for any of the attributes of the metric. For example, you can choose not to send the `host.process.cpuPercent` of all processes whose `process.name` match the `^java` [regular expression](https://github.com/google/re2/wiki/Syntax).
+
+    In this example, we exclude process metrics using executable files and names:
+
+    ```
+    exclude_matching_metrics: # You can combine attributes from different metrics
+        process.name:
+          - regex "^java"    # Exclude all processes starting with "java"
+        process.executable:
+          - "/usr/bin/python2"              # Exclude the Python 2.x executable
+          - regex "\\System32\\svchost"     # Exclude all svchost executables
+    ```
+
+    To configure `exclude_matching_metrics` as an environment variable for the [Kubernetes integration](/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration/), add it in the manifest inside the `env:` object:
+
+    ```yml
+    env:
+        - name: NRIA_EXCLUDE_MATCHING_METRICS
+          value: |
+            process.name:
+              - regex "^java"
+            process.executable:
+              - "/usr/bin/python2"
+              - regex "\\System32\\svchost"
+    ```
+
+    Default
+
+    <table>
+      <thead>
+        <tr>
+          <th style={{ width: "200px" }}>
+            YML option name
+          </th>
+
+          <th style={{ width: "200px" }}>
+            Environment variable
+          </th>
+
+          <th>
+            Type
+          </th>
+
+          <th>
+            Default
+          </th>
+
+          <th>
+            Version
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        <tr>
+          <td>
+            `exclude_matching_metrics`
+          </td>
+
+          <td>
+            `NRIA_EXCLUDE_MATCHING_METRICS`
+          </td>
+
+          <td>
+            metric.attribute:
+
+            \- regex "pattern"
+
+            \- "string"
+
+            \- "string-with-wildcard\*"
+          </td>
+
+          <td/>
+
```

Comment:
we can anticipate this will require [1.57.0](https://github.com/newrelic/infrastructure-agent/releases/tag/1.57.0)

Reply:
Done in https://github.com/newrelic/docs-website/pull/18505/commits/58f21277ce1b1c0a4c2f5f7992fa68c55215c5d8!

Reply:
I meant, the last column `Version` in this table afaik represents the agent version where this setting was introduced, so we can just have it there.

Reply:
Oops, sorry, correcting

Reply:
Done in https://github.com/newrelic/docs-website/pull/18505/commits/757024434ae257dc9c3071c34d35dba7129a711e!